### PR TITLE
Add Lyric View button

### PR DESCRIPTION
**Pull request by brandonxia01**
Added a button on the top tool bar that switches between Image view and Lyric only view.
Only the 1st verse (of some songs) appears so far.

We could definitely use some UX and HTML help here :^)

ezgif-7-010da9d8b6e7

### CODE REVIEW

File: src/components/LyricViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <b>
+          {/* I have no idea how to print these lyrics! 
+          
+          fyi - a couple of the songs use capitals verse numbers for some reason (ie: V1 instead of v1)
+          */}
+        {data['lyrics']['v1']}
+      </b>
+
+      <footer>Author: {data['author'] == '' ? 'Unknown' : data['author']}</footer>
+    </div>
```
Comment (brandonxia01) [inline src/components/LyricViewer.tsx:31]: I couldn't figure out how to display all the verses properly here.

The lyrics are stored with each verse being a separate object and each line as part of an array within that verse. Tricky tricky.

File: src/pages/HomePage.tsx [hunk 2]
```diff
@@ -28,9 +52,15 @@ const HomePage: React.FC = () => {
...
-          </div>
+          lyricsOnlyMode ? (
+            <div id="lyricDiv">
+              <LyricViewer songNumber={number} />
+            </div>
+          ) : (
+            <div id="songDiv">
+              <SongViewer songNumber={number} />
+            </div>
+          )
```
Comment (brandonxia01) [inline src/pages/HomePage.tsx:63]: I've been using ternary operators to conditionally show and hide different components, this is probably going to get ugly fast if a couple more are added like this.

@slowhei any suggestions?

Comment (brotatotes) [reply src/pages/HomePage.tsx:63]: Could you do something like

`songVisibility && lyricsOnlyMode ? (...`

Comment (brotatotes) [reply src/pages/HomePage.tsx:63]: Maybe we can have them as separate IonContents and show 1 and hide the other at all times

**Review by brotatotes — APPROVED** [review]

File: src/components/LyricViewer.tsx (hunk 1 above)
Comment (brotatotes) [reply src/components/LyricViewer.tsx:31]: the `V1` vs `v1` thing will need to be corrected at some point.

File: src/pages/HomePage.tsx (hunk 2 above)
Comment (brandonxia01) [reply src/pages/HomePage.tsx:63]: I'll give that a try

**Pull request merged**